index.html

```diff
@@ -180,14 +189,16 @@ <h2 class="tagline">OK, but <strong>who's coming?</strong></h2>
         <div class="row">
             <div class="large-12 columns">
                 <h2>Ok, <strong>I'm in.</strong></h2>
-                <h4>Glad to hear it.  <!--<a target="_blank" href="https://goo.gl/forms/R5m8EEqf6rDyrTlf2">RSVP here</a>, then--> We'll see you there!</h4>
+                <h4>Glad to hear it.  <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSdjvDOa-lmXu4PnTFTyDGrKmtLU0MjIFThRQolVfrdiOxOHiA/viewform ">RSVP here</a>, then We'll see you there!</h4>
             </div>
         </div>
     </section>
     <section id="history">
         <div class="row">
             <div class="large-12 columns">
                 <h2>A storied tradition</h2>
+                <h4>At our <a href="2016_fall.html">Fall 2016 Hackathon</a> (photo at top), over 300 hackers joined us in Roone Arledge Auditorium to work on Lab 3, collect swag, eat food, and witness the unveiling of the midnight surprise---Insomnia cookies!  See more photos <a href="2016_fall_photos.html">here</a>.</h4>
```

Comment:
i don't know if the `---` will render well on in a web browser. do you think we could change it to a `: `?